### PR TITLE
Update EIP-5005: Move status to "Review"

### PR DESCRIPTION
**Pull request by auryn-macmillan**
This PR changes the status of ERC-5005 from "Draft" to "Review".

### CODE REVIEW

**Review by Pandapip1 — COMMENTED** [review]
NOTE: I am not done with this review. I accidentally hit ctrl+enter as I was writing a comment. I will be updating this!

I'm liking the look of this EIP. I would love to see the DAO use-cases for this!

Since I cannot make changes on unmodified lines (thanks GitHub!), here are my suggestions:

Line 15: Replace `ERC-5005 (Zodiac Avatar Accounts)` with `This EIP`
Line 15: This EIP is not a philosophy, unless you consider the question "how does one make a good contract account" one of the fundamental questions of the universe. In that case, go ahead :)
Line 18: This might be a good time to mention separation of concerns.
Line 20: Replace `Zodiac standard` with `this EIP`
Line 30: Replace `The Zodiac standard` with `This EIP`
Line 30: Replace `-` with `: `
Line 211: I would prefer if you expanded this line into multiple lines.
Line 216: I suggest this wording change: `The interface defined in this standard is designed to be mostly compatible with most popular programmable accounts in use right now, to minimize the need for changes to existing tooling.`
Line 222: Replace the `.` with a `:`
Line 223-224: Replace `unilateral` with `absolute`
Line 225: Remove `Be warned`. This entire section is supposed to be a big warning.
Line 225: `Replace `safe guard` with `safeguard`

File: EIPS/eip-5005.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ requires: 165
 ---
 
 ## Abstract
-ERC-5005 (Zodiac Avatar Accounts) is a philosophy and open standard for composable and interoperable tooling for programmable Ethereum accounts. Zodiac-compatible tooling separates the account taking actions/holding tokens (known as the "avatar") and the authorization logic into two (or more) separate contracts. This standard defines the `IAvatar` interface, to be implemented by avatar contracts, while the authorization logic can be implemented with any combination of other tools (for example, DAO tools and frameworks).
+ERC-5005 (Zodiac Modular Accounts) is an open standard for composable and interoperable tooling for programmable Ethereum accounts. Compatible tooling separates the account taking actions/holding tokens (known as the "avatar") and the authorization logic into two (or more) separate contracts. This standard defines the `IAvatar` interface, to be implemented by avatar contracts, while the authorization logic can be implemented with any combination of other tools (for example, DAO tools and frameworks).
```
Comment (Pandapip1) [inline EIPS/eip-5005.md]:
```suggestion
This EIP (Zodiac Modular Accounts) is an open standard for composable and interoperable tooling for programmable Ethereum accounts. Compatible tooling separates the account taking actions/holding tokens (known as the "avatar") and the authorization logic into two (or more) separate contracts. This standard defines the `IAvatar` interface, to be implemented by avatar contracts, while the authorization logic can be implemented with any combination of other tools (for example, DAO tools and frameworks).
```

Comment (auryn-macmillan) [reply EIPS/eip-5005.md]: The "commit suggestion" button on GitHub didn't work, so I just pushed the changes.

File: EIPS/eip-5005.md [hunk 2]
```diff
@@ -1,23 +1,24 @@
 ---
 eip: 5005
-title: Zodiac Avatar Accounts
+title: Zodiac Modular Accounts
 description: A composable design philosophy for programmable accounts.
```
Comment (Pandapip1) [inline EIPS/eip-5005.md:4]:
```suggestion
description: Composable interoperable programmable accounts
```

**Comment by Pandapip1** [comment]
@auryn-macmillan mind reviewing my changes?

**Comment by auryn-macmillan** [comment]
Looks good to me

**Review by Pandapip1 — APPROVED** [review]

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 5005 state from Draft to Review

**Pull request merged**